Companion app as home app (#1268)
home-assistant/companion.home-assistant: docs/integrations/android-home-app-launcher.md

---
title: "Android Home App (launcher)"
id: 'android-home-app-launcher'
---

![Android](/assets/android.svg)

The Home Assistant Companion app can be set as your device's default home app (launcher). This turns your Android device into a dedicated Home Assistant control panel, perfect for wall-mounted tablets or kiosks.

> ## What is home app?

Reviewer: It isn't a name, it's a description of a role. You set a home app/launcher on the device.

A home app (also called a "launcher") is the app that appears when you press the home button. By setting Home Assistant as your home app, your device boots directly into Home Assistant, making it ideal for dedicated smart home control.

## Use cases

- Wall-mounted tablets: Create a dedicated control panel for your home.
- Kitchen displays: Quick access to timers, shopping lists, and home controls.
- Bedside tablets: Control lights, alarms, and media from your nightstand.
- Kiosk mode: Set up a device for guests or family members with limited access.
- Digital signage: Display dashboards, cameras, or information screens.

## What to expect

When Home Assistant is your default home app:

- No app drawer: You won't see the usual list of installed apps when pressing the home button.
- No direct app launching: You can't start other apps the traditional way.
- Home button: Returns to Home Assistant (or refreshes if already open).
- Back button: When the navigation stack is empty, pressing back will refresh the app instead of doing nothing.
- Boot: The device opens Home Assistant automatically after restarting.

> ## Enabling home app

Reviewer 2: ```suggestion
## Enabling Home Assistant as home app
```

> ## Enabling Home Assistant as home app

Reviewer 2: ```suggestion
## Enabling Home Assistant as the home app
```

1. Open the Home Assistant app.
2. Go to [**Settings**](https://my.home-assistant.io/redirect/config/) > **Companion app**.
3. Scroll down to **Device home screen**.
4. Enable **Use as Home app (launcher)**.

    <img alt="Screenshot showing allow as home app" src="/assets/android/allow_as_home_app.png" width='400'/>

5. To open the system screen that allows you to pick a home app, select **Change home app**.

    <img alt="Screenshot showing choose a home app system screen" src="/assets/android/select_home_app.png" width='400'/>

> ## Disabling Home Assistant as home app

Reviewer 2: ```suggestion
## Disabling Home Assistant as the home app
```

If you want to switch back to your regular home app:

1. Open the Home Assistant app.
2. Go to [**Settings**](https://my.home-assistant.io/redirect/config/) > **Companion app**.
3. Scroll down to **Device home screen**.
4. To open the system screen that allows you to pick a home app other than Home Assistant, select **Change home app**.

    <img alt="Screenshot showing choose a home app system screen" src="/assets/android/select_home_app.png" width='400'/>

## Creating a kiosk mode setup

For the best kiosk or wall tablet experience, consider these additional settings:

### Enable local push

Enable [local push](/docs/notifications/notification-local) to take advantage of this:

- Unlimited notifications: No daily limits on alerts from your automations
- Unlimited commands: No daily limits on controlling the device (change volume, open apps)

> - Faster response: Commands run instantly with lower latency

Reviewer: ```suggestion
- Faster response: Notifications and commands arrive and run with lower latency
```

- Works offline: Keeps working even when your internet is down

### Keep screen on

> To prevent the screen from turning off, go to [**Settings**](https://my.home-assistant.io/redirect/config/) > **Companion App** > **Other Settings** and enable **Keep screen on**.

Reviewer: Other settings isn't an item you can select , but a heading - this should be "scroll down to" like the previous settings description and sentence case.
